File: fretecalc/app/src/main/java/com/example/fretecalc/models/routes/Point.java

package com.example.fretecalc.models.routes;

import com.google.gson.annotations.Expose;

import java.util.List;

public class Point {

    @Expose
    private List<Double> point;
    @Expose
    private String provider;

    public List<Double> getPoint() {
        return point;
    }

    public void setPoint(List<Double> point) {
        this.point = point;
    }

    public String getProvider() {
        return provider;
    }

    public void setProvider(String provider) {
        this.provider = provider;
    }
}

File: app/src/main/java/com/truckpadcase/calculatefreight/domain/model/remote/Point.kt
package com.truckpadcase.calculatefreight.domain.model.remote

import com.google.gson.annotations.SerializedName
import java.io.Serializable


data class Point (

    @SerializedName("point")
    val point: List<Double>,

    @SerializedName("provider")
    val provider : String
) : Serializable